templates/python/LocalPubsub/test_LocalPubSub/test_pub.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+from localpubsub import pub
+
+
+@pytest.fixture
+def ipc_client(mocker):
+    mocker.patch("awsiot.greengrasscoreipc.clientv2.GreengrassCoreIPCClientV2")
+    mocker.patch("awsiot.greengrasscoreipc.connect")
+    from awsiot.greengrasscoreipc.clientv2 import GreengrassCoreIPCClientV2
+
+    client = GreengrassCoreIPCClientV2()
+    return client
+
+
+def test_publish_message_N_times(ipc_client):
+    pub.publish_message_N_times(ipc_client, "topic", "World")
+    assert ipc_client.publish_to_topic.call_count == 10
```

Comment:
We should strive to make this sample test a bit better. Asserting call count doesn't tell us things are working. Maybe assert that it was called with what you expect